AmplifyPlugins/Auth/AWSCognitoAuthPluginIntegrationTests/AuthSessionTests/SignedInAuthSessionTests.swift

```diff
@@ -15,7 +15,7 @@ import AmplifyTestCommon
 class SignedInAuthSessionTests: AWSAuthBaseTest {
 
     override func setUpWithError() throws {
-        try initializeAmplifyWithError()
+        try initializeAmplify()
```

Comment:
I created `initializeAmplifyWithError()` in https://github.com/aws-amplify/amplify-swift/pull/2693

Now, I'm collapsing `initializeAmplify()` and `initializeAmplifyWithError()` into one.